docs/faq/help.txt

```diff
@@ -5,15 +5,34 @@ FAQ: Getting Help
 How do I do X? Why doesn't Y work? Where can I go to get help?
 ==============================================================
 
-If this FAQ doesn't contain an answer to your question, you might want to
-try the |django-users| mailing list. Feel free to ask any question related
-to installing, using, or debugging Django.
+First, please check if your question is answered on the
+:doc:`FAQ index <faq/index>`. Please also search for answers using your
+favorite search engine, and in `the forum`.
 
-If you prefer IRC, the `#django IRC channel`_ on the Freenode IRC network is an
-active community of helpful individuals who may be able to solve your problem.
+.. _`the forum`: https://forum.djangoproject.com/
 
+If you can't find an answer, please take a few minutes to formulate your
+question well. Explaining the problems you are facing clearly will help others
+help you. See the StackOverflow guide on `asking good questions`.
+
+.. _`asking good questions`: https://stackoverflow.com/help/how-to-ask
+
+Then, please post it in one of the following channels:
+
+* The Django Forum section `"Using Django"`_. This is for web-based
+  discussions.
+* The |django-users| mailing list. This is for email-based discussions.
+* The `#django IRC channel`_ on the Freenode IRC network. This is for
+  chat-based discussions.
+
+.. _`"Using Django"`: https://forum.djangoproject.com/c/users
 .. _`#django IRC channel`: irc://irc.freenode.net/django
 
+In all these channels, please be friendly and patient, and otherwise abide by
```

Comment:
I'm not so sure about ',and otherwise'. My view is it would be ok without the word 'otherwise'.
I think the oxford comma helps readability here.

Reply:
"Be friendly and patient" is the first point in the Code of Conduct, so it's a separate clause to "and otherwise..". I've rewritten it to split a slightly longer summary of the code afterwards, for those readers who won't click the link.